src/helper.ts

```diff
@@ -211,6 +211,12 @@ class Helper {
     tokens.push('$');
     return new RegExp(tokens.join(''));
   }
+
+  static completeUserURL(urlString: string): string {
+    if (urlString.startsWith('localhost') || urlString.startsWith('127.0.0.1'))
```

Comment:
Perhaps allow `0:0:0:0:0:0:0:1` and `::1` as well?

Reply:
I'd strongly encourage using URLs where urls are required. localhost and 127.0.0.1 are too wide-spread to ignore, but ipv6 is not (yet).